cms/src/main/java/uk/nhs/digital/apispecs/services/RemoteSpecService.java

```diff
@@ -16,21 +13,23 @@
 import java.util.List;
 import java.util.function.Supplier;
 
-public class ApigeeService implements OpenApiSpecificationRepository {
 
-    private static final Logger log = LoggerFactory.getLogger(ApigeeService.class);
+public abstract class RemoteSpecService implements OpenApiSpecificationRepository {
 
-    private static final String RESOURCE_NAMESPACE_APIGEE_MANAGEMENT_API = "apigeeManagementApi";
+    private static final Logger log = LoggerFactory.getLogger(RemoteSpecService.class);
```

Comment:
```suggestion
    private static final Logger log = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
```

If you use this then the log will show the extending class name (i.e. ProxygenService) rather than this abstract one.

Reply:
I tried doing this but the logs were still coming up as originating from parent class (RemoteSpecService). I even tried passing the instance of the Logger to the parent through the constructor but still no luck 🤔 
![image](https://user-images.githubusercontent.com/91538797/174332600-ea573fc7-cc90-407d-9e6b-c19cecaa1c7b.png)

Reply:
As a work around I have added the extending class name to the log text to make it clearer :)

<img width="1265" alt="image" src="https://user-images.githubusercontent.com/91538797/174617753-0d43b908-07d7-4baa-b2fc-51476c989bec.png">